File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    gop = 1
    hap = 0
    
    for i in num_list:
        gop *= i
        hap += i
        
    hap = hap**2
    
    if gop < hap:
        return 1
    else:
        return 0

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int gop = num_list[0];
        int hapje = num_list[0];
        
        for(int i=1; i<num_list.length; i++){
            gop *= num_list[i];
            hapje += num_list[i];
        }
        
        if(gop < hapje*hapje) {return 1;}
        else { return 0; }
    }
}